<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>69</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_choose_file">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>489</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_rotation_z">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>59</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string>Z: </string>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_translations_x">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>9</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string>X: </string>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_rotation_y">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>59</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string>Y: </string>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="MyOpenGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 37, 34)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_translations_y">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>9</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string>Y: </string>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_rotation_x">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>59</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string>X: </string>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>19</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Перемещение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>119</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштаб</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_translations_z">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>9</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string>Z: </string>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>1002</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Имя файла:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_filename">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="overwriteMode">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Вершин:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_edges_count">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Ребер:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_verticles_count">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDoubleSpinBox" name="scale">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> Цвет фона</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> Цвет линий</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> Цвет точек</string>
    </property>
   </widget>
   <widget class="QSlider" name="color_back_red">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="color_back_green">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="color_back_blue">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="color_line_red">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="color_line_green">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="color_line_blue">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>240</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="color_dot_red">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="color_dot_green">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="color_dot_blue">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер точек</string>
    </property>
   </widget>
   <widget class="QSlider" name="size_dot">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>340</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер линий</string>
    </property>
   </widget>
   <widget class="QSlider" name="size_lin">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>390</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>430</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отображение линий</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Отображение точек</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nothingdot">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Не отображать</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="stripe_line">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>460</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Штрих-пунктирные линии</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dot_circle">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Окружность</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dot_square2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Квадрат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jpeg">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.jpeg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bmp">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>490</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.bmp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gif">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>490</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>.gif</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
